tools/tox_matrix.py

```diff
@@ -144,14 +142,6 @@ def get_matrix_item(env, global_libraries, global_string_parameters,
     else:
         item["python_version"] = env.get("default_python") or default_python
 
-    # if Python is <3.10 we can't use macos-latest which is arm64
-    try:
-        if Version(item["python_version"]) < Version('3.10') and item["os"] == "macos-latest":
-            item["os"] = "macos-13"
```

Comment:
I thought about not removing this for a while, but actually I think removing it and having the error be "can't find Python 3.9" is better than "huh what why is this on macos-13".

Reply:
:shrug: lol